docs/app/Examples/elements/Label/Types/LabelExampleAttached.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { Grid, Image, Label, Segment } from 'stardust'
+
+const imageStyles = {
```

Comment:
Are these style required? I'd like to avoid any styles in the docs. It would seem that SUI alone should handle all styling.